src/main/webapp/app/entities/file-upload-exercise/file-upload-exercise.module.ts

```diff
@@ -7,39 +7,32 @@ import { ArtemisSharedModule } from 'app/shared';
 import {
     FileUploadExerciseComponent,
     FileUploadExerciseDeleteDialogComponent,
-    FileUploadExerciseDeletePopupComponent,
     FileUploadExerciseDetailComponent,
-    FileUploadExerciseDialogComponent,
-    FileUploadExercisePopupComponent,
-    fileUploadExercisePopupRoute,
     FileUploadExercisePopupService,
     fileUploadExerciseRoute,
     FileUploadExerciseService,
     FileUploadExerciseUpdateComponent,
 } from './';
 import { SortByModule } from 'app/components/pipes';
+import { FormDateTimePickerModule } from 'app/shared/date-time-picker/date-time-picker.module';
+import { ArtemisCategorySelectorModule } from 'app/components/category-selector/category-selector.module';
+import { ArtemisDifficultyPickerModule } from 'app/components/exercise/difficulty-picker/difficulty-picker.module';
+import { ArtemisMarkdownEditorModule } from 'app/markdown-editor';
 
-const ENTITY_STATES = [...fileUploadExerciseRoute, ...fileUploadExercisePopupRoute];
+const ENTITY_STATES = [...fileUploadExerciseRoute];
 
 @NgModule({
-    imports: [ArtemisSharedModule, RouterModule.forChild(ENTITY_STATES), SortByModule],
-    declarations: [
-        FileUploadExerciseComponent,
-        FileUploadExerciseDetailComponent,
-        FileUploadExerciseUpdateComponent,
-        FileUploadExerciseDialogComponent,
-        FileUploadExerciseDeleteDialogComponent,
-        FileUploadExercisePopupComponent,
-        FileUploadExerciseDeletePopupComponent,
-    ],
-    entryComponents: [
-        FileUploadExerciseComponent,
-        FileUploadExerciseUpdateComponent,
-        FileUploadExerciseDialogComponent,
-        FileUploadExercisePopupComponent,
-        FileUploadExerciseDeleteDialogComponent,
-        FileUploadExerciseDeletePopupComponent,
+    imports: [
+        ArtemisSharedModule,
+        RouterModule.forChild(ENTITY_STATES),
+        SortByModule,
+        FormDateTimePickerModule,
+        ArtemisCategorySelectorModule,
+        ArtemisDifficultyPickerModule,
+        ArtemisMarkdownEditorModule,
     ],
+    declarations: [FileUploadExerciseComponent, FileUploadExerciseDetailComponent, FileUploadExerciseUpdateComponent, FileUploadExerciseDeleteDialogComponent],
+    entryComponents: [FileUploadExerciseComponent, FileUploadExerciseUpdateComponent, FileUploadExerciseDeleteDialogComponent],
```

Comment:
Please remove all components here, that are not opened in a dialog anymore.

Reply:
Done